template-processors/openshift-provision/bin/processParameters.sh

```diff
@@ -19,5 +19,5 @@ set -euxo pipefail
 echo "Processing parameters in ${CLONED_PARAMETER_GIT_DIR}"
 
 ansible-playbook /files/processParameters.yml \
-    ${TEMPLATE_PROCESSOR_ARGS:-} \
+    "${TEMPLATE_PROCESSOR_ARGS:-}" \
```

Comment:
change adding `"` here is not correct, we need contents of the env variable to be passed to command as separate arguments